IFComp/root/src/_change_vote.tt

```diff
@@ -1,5 +1,5 @@
 <script>
```

Comment:
As discussed on Slack, please revert this change back to class-based element selection. Move the JS block into a `[% BLOCK scripts %]` TT block, and wrap in a jquery `$( document ).ready(function() { <PUT CODE HERE> });`.